yt_dlp/extractor/zdf.py

```diff
@@ -202,8 +206,8 @@ def _extract_entry(self, url, player, content, video_id):
         ptmd_path = t.get('http://zdf.de/rels/streams/ptmd')
 
         if not ptmd_path:
-            ptmd_path = t[
-                'http://zdf.de/rels/streams/ptmd-template'].replace(
+            ptmd_path = traverse_obj(
+                t, ('streams', 'default', 'http://zdf.de/rels/streams/ptmd-template')).replace(
```

Comment:
why use traverse_obj here? The point of it is to handle missing data without throwing errors. If the keys are absent, the .replace will throw error anyway. Also, further extraction cannot continue without this data

Reply:
actually, this would be better, no?
```suggestion
            ptmd_path = traverse_obj(
                t, ('streams', 'default', 'http://zdf.de/rels/streams/ptmd-template'),
                'http://zdf.de/rels/streams/ptmd-template').replace(
```

Reply:
Yup, querying both places seems like a safer bet